### ISSUE TITLE
Abbreviations do not have expansions like in WCAG

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#690]
@daniel-montalvo I noticed that one of the editorial changes in the December WCAG 2.2 update was that anywhere "HTML" is used, markup was added to have the abbreviation expanded on hover to "HyperText Markup Language". When I looked into the code, it seems that the `<abbr>` markup is probably added via a post document build script - not in the source code. 

Wondering if we should have a similar script run to add tags to WCAG2ICT abbreviations for HTML or if we should leave it out.

**Comment by bruce-usab** [comment]
Thank you @maryjom for this level of detailed consideration!

**Comment by daniel-montalvo** [comment]
Hi @kfranqueiro 

Could you please have a look at @maryjom above?

Is there any post processing  task that is adding `abbr` to HTML in AGWG new scripts?

Thanks.

**Comment by kfranqueiro** [comment]
This turns out to be a [ReSpec feature](https://respec.org/docs/#abbreviations), not specific to w3c/wcag's processing. There is exactly one instance of `<abbr title="HyperText Markup Language">HTML</abbr>` within the document source; ReSpec finds this, then goes searching for other instances and wraps them.

I can reproduce the same behavior within WCAG2ICT: if I manually wrap any one of the occurrences of "HTML" in `comments-by-guideline-and-success-criterion.md`, ReSpec wraps the rest.

**Comment by daniel-montalvo** [comment]
Thanks  @kfranqueiro 

Unfortunately our scripts were conceived to run after Respec. I am assuming for this feature to be leveraged by the WCAG2ICT project we would probably need to update them so that they run before.

@maryjom I also wonder how much of a priority this is given the inconsistent support of `abbr` after all.

I'll see if  I can work on #708

**Comment by maryjom** [comment]
This is done as much as is possible for now. WCAG 2.1 quoted content using HTML doesn't have the abbreviation because it is pulled in and they aren't currently doing the post-processing to add it yet.  This will naturally fix itself if and when the WCAG 2.1 build changes to do this.
